### ISSUE TITLE
[INSTALL ON LINUX] No "Spicetify" or "CustomApps" folder

### ISSUE THREAD

**Issue by skal-dB** [harbassan/spicetify-apps#153]
1/ Here are the instrcutions I have through the Marketplace : 

> _Download the zip file in the_ [latest release](https://github.com/harbassan/spicetify-apps/releases?q=library&expanded=true), _rename the unzipped folder to library, then place that folder into your `CustomApps` folder in the `spicetify` directory and you're all done._ 
> 
> _If everything's correct, the structure should be similar to this:_
> 
> 📦spicetify\CustomApps
>  ┣ 📂marketplace
>  ┣ etc...
>  ┗ 📂library
>  ┃ ┣ 📜extension.js
>  ┃ ┣ 📜index.js
>  ┃ ┣ 📜manifest.json
>  ┃ ┗ 📜style.css

2/ So I'm looking for the _`CustomApps`_ folder and _`Spicetify`_ directory on my computer but they do not exist

3/ The only things I can find with _Spicetify_ in the name are files located in _`/usr/share/spotify/Apps/xpui`_

Any idea on how to solve this ?

**Comment by bombboy4O** [comment]
Hello, did you go through AppData\Local\spicetify\CustomApps directory? Press windows+r to bring up Run dialog box and type appdata.

**Comment by cyjico** [comment]
Should be in `%appdata%\spicetify\CustomApps\`. Though, you can run this script as well:
```ps1
iwr -useb "https://raw.githubusercontent.com/cyjico/spicetify-apps/refs/heads/auto-install-win/stats/install.ps1" | iex
```
(from PR https://github.com/harbassan/spicetify-apps/pull/154)

**Comment by cyjico** [comment]
Try here instead:

![image](https://github.com/user-attachments/assets/06113d3c-ea16-4600-8490-fc02f6cf0b8d)

**Comment by jacobtizel** [comment]
Yes - my spotify directory was also under the ~/.config directory. Would reccomend checking there.
